src/analysis/normed_space/basic.lean

```diff
@@ -720,6 +720,9 @@ namespace real
 
 lemma norm_eq_abs (r : ℝ) : ∥r∥ = abs r := rfl
 
+lemma norm_of_nonneg {x : ℝ} (hx : 0 ≤ x) : ∥ x ∥ = x :=
```

Comment:
```suggestion
lemma norm_of_nonneg {x : ℝ} (hx : 0 ≤ x) : ∥x∥ = x :=
```
(I think this is current style?  Feel free to change back if you prefer otherwise.)

Reply:
You are right, that is the current style. I didn't know that, so thanks for pointing that out!